servers/rendering/renderer_scene_render.h

```diff
@@ -35,6 +35,8 @@
 #include "core/templates/paged_array.h"
 #include "servers/rendering/renderer_storage.h"
 
+#define RENDER_MAX_VIEWS 2
```

Comment:
After discussing with @reduz , see if we can add this to the enums, maybe part of `RendererCompositor` so it's cleaner. I had problems with this before but may have been due to unrelated issues.